cdap-api/src/main/java/co/cask/cdap/api/security/store/SecureStore.java

```diff
@@ -29,15 +29,21 @@
 public interface SecureStore {
 
   /**
+   * List of all the entries in the secure store.
    * @param namespace The namespace that this key belongs to.
    * @return A list of {@link SecureStoreMetadata} objects representing the data stored in the store.
+   * @throws IOException If there was a problem reading from the keystore.
+   * @throws Exception If the specified namespace does not exist.
```

Comment:
Yes, those exceptions are in cdap-common and are not exposed.


> On Aug 2, 2016, at 2:35 PM, Bhooshan Mogal <notifications@github.com> wrote:
> 
> In cdap-api/src/main/java/co/cask/cdap/api/security/store/SecureStore.java <https://github.com/caskdata/cdap/pull/6274#discussion_r73242605>:
> 
> >     * @param namespace The namespace that this key belongs to.
> >     * @return A list of {@link SecureStoreMetadata} objects representing the data stored in the store.
> > +   * @throws IOException If there was a problem reading from the keystore.
> > +   * @throws Exception If the specified namespace does not exist.
> So we don't mention NamespaceNotFoundException here because this is API?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/caskdata/cdap/pull/6274/files/b5c602cb63a60e61be6506a3072b89a5be3e40aa#r73242605>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AAFGkz_gt9LWi4Lu7VYPFqvMgCxAb6HFks5qb7g9gaJpZM4JZI-3>.
>